OOP/OOPS_Session_1/ServiceStation.java

```diff
@@ -0,0 +1,158 @@
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.util.*;
+class Mechanics
+{
+	private String mechanicName,specializeCar;
+	private int rate;
+	public Mechanics(String mechanicName,String specializeCar,int rate)//Mechanics class constructor 
+	{
+		this.mechanicName=mechanicName;
+		this.specializeCar=specializeCar;
+		this.rate=rate;
+	}
+	public String getMName()//get method is used to return the value
+	{
+		return mechanicName;
+	}
+	public void set_M_Name(String mechanicName)//set method is used to store the value
+	{
+		this.mechanicName=mechanicName;
+	}
+	public void setSCar(String specializeCar)
+	{
+		this.specializeCar=specializeCar;
+	}
+	public String getSCar()
+	{
+		return specializeCar;
+	}
+	public void setServiceRate(int rate)
+	{
+		this.rate=rate;
+	}
+	public int getServiceRate()
+	{
+		return rate;
+	}
+	public String toString()//to return complete information of Mechanics
+	{
+		return " "+mechanicName+" "+specializeCar+" "+rate;
+	}
+}
+class Car_Type
+{
+	private String carName,carNumber;
+	public Car_Type(String carNumber,String carName)
+	{
+		this.carName=carName;
+		this.carNumber=carNumber;
+	}
+	public String getCarName()
+	{
+		return carName;
+	}
+	public void setCarName(String carName)
+	{
+		this.carName=carName;
+	}
+	public String getCarNum()
+	{
+		return carNumber;
+	}
+	public void setCarNum(String carName)
+	{
+		this.carNumber=carName;
+	}
+	public String toString()//to return complete information of Car
+	{
+		return " "+carNumber+" "+carName;
+	}
+}
+public class ServiceStation 
+{
+	public static void main(String args[])throws FileNotFoundException
+	{
+		Scanner fin_Mechanic=new Scanner(new FileReader("Mechanics.txt"));//to read information of Mechanics from given file
+		String mechanicName,specializeCar;
+		int num_of_mechanics=0;
+		int rate;
+		int countSuv=0,countSedan=0,countHatchbak=0;
+		int m_countSuv=0,m_countSedan=0,m_countHatchbak=0;
+		
+		Scanner fin_Car=new Scanner(new FileReader("Car.txt"));
+		String carName,carNumber;
+		int num_of_car;
+		num_of_car=fin_Car.nextInt();//get the first line of file(total number of cars)
+		String temp;
+		Car_Type car[]=new Car_Type[num_of_car];
+		for(int i=0;i<num_of_car;i++)//to store complete information of car for all object
```

Comment:
ACTION : main method shouldn't contain this much code. Break down this code into multiple functions.